assets/js/theme/global/modal.js

```diff
@@ -17,6 +17,7 @@ export const ModalEvents = {
     closed: 'closed.fndtn.reveal',
     open: 'open.fndtn.reveal',
     opened: 'opened.fndtn.reveal',
+    loaded: 'loaded.data.custom',
```

Comment:
Question: Why opened event does not satisfy our needs?

Reply:
when opened fired modal opened but content still can not be loaded and there are no element user can tab through